lib/l4p/tcp_rxtx.c

```diff
@@ -303,17 +304,18 @@ tcp_fill_mbuf(struct rte_mbuf *m, const struct tle_tcp_stream *s,
  *  - if no HW cksum offloads are enabled, calculates TCP checksum.
  */
 static inline void
-tcp_update_mbuf(struct rte_mbuf *m, uint32_t type, const struct tcb *tcb,
+tcp_update_mbuf(struct rte_mbuf *m, uint32_t type, const struct tle_tcp_stream *s,
 	uint32_t seq, uint32_t pid, uint8_t tcp_flags)
 {
+	const struct tcb *tcb = &s->tcb;
```

Comment:
Same question as above.

Reply:
Same answer - used but out of diff view.